data/yaml/page-furniture/sidebar.yaml

```
@@ -160,8 +160,14 @@ items:
       - label: 'Ably accounts and apps'
         link: ''
         items:
-          - label: 'Account overview'
-            link: /account
+          - label: 'User management'
+            link: /account/users
+          - label: 'Organizations'
+            link: /account/organizations
+          - label: 'Single sign-on (SSO)'
+            link: /account/sso
+          - label: 'Two-factor authentication (2FA)'
+            link: /account/2fa
```

```suggestion
          - label: 'Account overview'
            link: /account
          - label: 'User management'
            link: /account/users
          - label: 'Organizations'
            link: /account/organizations
          - label: 'Single sign-on (SSO)'
            link: /account/sso
          - label: 'Two-factor authentication (2FA)'
            link: /account/2fa
```

This was removing the overview page.